nilearn/input_data/tests/test_multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+
+import nibabel
+import pytest
+
+from nilearn.input_data.nifti_labels_masker import NiftiLabelsMasker
+from nilearn.input_data.multi_nifti_labels_masker import MultiNiftiLabelsMasker
+from nilearn._utils import testing, as_ndarray, data_gen
+from nilearn._utils.exceptions import DimensionError
+from nilearn.image import get_data
+
+
+def generate_random_img(shape, length=1, affine=np.eye(4),
```

Comment:
There are already functions in Nilearn to generate fake data. Use them instead of writing a new one.

Reply:
Done!